src/test/java/org/opentripplanner/middleware/ApiKeyManagementTest.java

```diff
@@ -61,13 +63,23 @@ public static void setUp() throws IOException, InterruptedException {
      */
     @AfterAll
     public static void tearDown() {
-        assumeTrue(isEndToEnd);
-        setAuthDisabled(getDefaultAuthDisabled());
-        // Delete admin user.
-        Persistence.adminUsers.removeById(adminUser.id);
+        assumeTrue(isEndToEndAndAuthIsDisabled());
+        // Note: apiUser.delete() cannot be used because no Auth0 user is created. The method fails on attempting to
+        // delete the random value assigned to the apiUser on initialization. This in turn prevents the apiUser from
+        // being delete from Mongo.
```

Comment:
I think we should just overload the method with a boolean to optionally skip deleting on Auth0 user rather than copy-pasta delete code all over the place.

Reply:
Updated.